wdl-lint/tests/lints/between-import-whitespace/source.errors

```diff
@@ -51,3 +51,11 @@ warning[Whitespace]: more than one blank line in a row
    │  
    = fix: remove the unnecessary blank lines
 
+note[DescriptionMissing]: description key missing in meta section
+   ┌─ tests/lints/between-import-whitespace/source.wdl:22:5
+   │
+22 │     meta {}
+   │     ^^^^
+   │
+   = fix: add a description to the meta section
+
```

Comment:
Could we add an `#@ except: DescriptionMissing` to the existing files to suppress this lint and also add a test dedicated to intentionally missing a description?